tests/integration/incremental_unique_id_test/test_incremental_unique_id.py

```diff
@@ -4,8 +4,8 @@
 from pathlib import Path
 
 
-TestResults = namedtuple(
-    'TestResults',
+ResultHolder = namedtuple(
```

Comment:
Haha I actually suggested Matt to convert this to the same name

Reply:
I saw that and used the same name :). Pytest was also trying to collect "TestResults" as a test class too.